examples/common/example_scope_time.cpp

```diff
@@ -48,7 +48,7 @@ main ()
 {
   pcl::ScopeTime scope_time ("Test loop");
   {
-    float total;
+    
```

Comment:
This will optimize the loop out. See [godbolt](https://godbolt.org/z/M93cb-) for a solution to keep the busy loop